backends/fs/amigaos/amigaos-fs.cpp

```diff
@@ -384,7 +384,23 @@ Common::SeekableWriteStream *AmigaOSFilesystemNode::createWriteStream() {
 }
 
 bool AmigaOSFilesystemNode::createDirectory() {
-	warning("AmigaOSFilesystemNode::createDirectory() -> Not supported");
+	BPTR lock = IDOS->Lock(_sPath.c_str(), SHARED_LOCK);
+	if (lock) {
+		IDOS->UnLock(lock);
+		debug("AmigaOSFilesystemNode::createDirectory() -> Directory '%s' already exists!", _sPath.c_str());
+		return _bIsValid && _bIsDirectory; // TODO: Should we make sure it's really a directory?
```

Comment:
I don't understand this comment. Is there a possibility that a node with _bIsDirectory = true is not a directory?